src/build.rs

```diff
@@ -187,15 +199,26 @@ fn run_xbuild(args: &[String]) -> io::Result<process::ExitStatus> {
     Ok(exit_status)
 }
 
-fn create_kernel_info_block(kernel_size: u64) -> KernelInfoBlock {
+fn create_kernel_info_block(kernel_size: u64, maybe_package_size: Option<u64>) -> KernelInfoBlock {
     let kernel_size = if kernel_size <= u64::from(u32::max_value()) {
         kernel_size as u32
     } else {
         panic!("Kernel can't be loaded by BIOS bootloader because is too big")
     };
 
+    let package_size = if let Some(size) = maybe_package_size {
+        if size <= u64::from(u32::max_value()) {
+            size as u32
+        } else {
+            panic!("Package can't be loaded by BIOS bootloader because is too big")
+        }
+    } else {
+        0
+    };
+
     let mut kernel_info_block = [0u8; BLOCK_SIZE];
     LittleEndian::write_u32(&mut kernel_info_block[0..4], kernel_size);
+    LittleEndian::write_u32(&mut kernel_info_block[8..12], package_size);
```

Comment:
Why `8..12` and not `4..8`?

Reply:
While the written out kernel size is 32bits, the read kernel size in the bootloader is 64bits. Thus, the written package_size here must not intersect with that. I didn't really want to mess with that.

Reply:
Oh, this seems like a bug then! We should probably create a crate analogous to os_bootinfo to keep these in sync.